server/models/Product.js

```diff
@@ -3,7 +3,7 @@ const mongoose = require('mongoose');
 const { Schema } = mongoose;
 
 const productSchema = new Schema({
-  name: {
+  productName: {
```

Comment:
We should check the front-end code to see how this will impact the current functionality. I'm pretty sure we can make this whole store as-is work with our design. Then we can add our styling.

Reply:
I'm fine with the change in the blue area. I needed to change name to productName in order  to differentiate and not get redundant and to match everything up from model to seeder (product.json)